package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
```

Comment:
You seem to be missing your `build` (run webpack for production) and `dev`(run webpack for dev) scripts.
Please add them.

Reply:
Please see [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png)

While you have added the `start` script we are more looking for `dev` and `build`